docs/conf.py

```diff
@@ -305,7 +305,7 @@
                 os.path.join(curdir, 'source/notebooks'))
 
 if 'readthedocs.org' in curdir:  # write config for readthedocs.org
-    with open(os.path.join(os.path.expanduser('~/.pyiron')), 'w') as f:
+    with open(os.path.join(os.path.expanduser('~/.pyiron_atomistic')), 'w') as f:
```

Comment:
I guess the name of the config file remains `~/.pyiron`.

Reply:
Yes, it absolutely should!